build/git-history.js

```diff
@@ -42,17 +42,22 @@ function getFromGit(contentRoot = CONTENT_ROOT) {
   // it seems `git log` prefers to use a newline character.
   // At least as of git version 2.28.0 (Dec 2020). So let's split on both
   // characters to be safe.
+  const parents = new Map();
   for (const line of output.split(/\0|\n/)) {
     if (line.startsWith(MARKER)) {
       const data = line.replace(MARKER, "").split(DELIMITER);
       hash = data[0];
       date = new Date(data[1]);
+      if (data[2]) {
+        const [, parentHash] = data[2].split(" ");
```

Comment:
@peterbe I think my previous comment was misleading. `data[2].split(" ")` has length 1 for any non-merge commit and therefore `parentHash` is `undefined` in those cases. I wanted to make this explicit  and only call `parents.set(parentHash, { modified: date, hash });` if `parentHash` is not `undefined`.

Reply:
Ok. I need to try to understand that. 
I've never been fully comfortable with how git trees work and the nomenclature it uses.